test/e2e/media.spec.ts

```diff
@@ -357,6 +373,16 @@ describe('Media Metadata endpoints', () => {
       expect(movie.searchMatches).toBeUndefined();
     });
 
+    it('should not return a movie result when looking for an episode', async() => {
```

Comment:
Would be good to add the inverse test that it returns the movie without the episode params

Reply:
That's a good point and it probably would. Will fix

Reply:
Good idea, that's done now